### ISSUE TITLE
Additional data preprocessing methods

### ISSUE THREAD

**Issue by bryanlimy** [alan-turing-institute/autoemulate#162]
- input data (and labels) to the model are currently standardized (per channel/feature)
- some model and data type benefit from different data preprocessing method, such as normalization (scale to [0, 1]).
- add an option to use different data preprocesing method, and allow it to be searched during hyperparameter optimization.

**Comment by bryanlimy** [comment]
I don't think this is urgent as whatever we have right now works? @mastoffel what do you think?

**Comment by mastoffel** [comment]
@bryanlimy @kallewesterling agree, not super urgent for now. But would ultimately be good to have.

**Comment by kallewesterling** [comment]
Sounds like we'll leave it out of the AI UK sprints then but keeping on the horizon. I've updated [the Miro board](https://miro.com/app/board/uXjVNGRNol4=/) accordingly.

**Comment by mastoffel** [comment]
closing as we can now use any `sklearn` [preprocessing method](https://scikit-learn.org/dev/api/sklearn.preprocessing.html)
